.concourse/pipeline.yaml.gomplate

```diff
@@ -364,7 +364,7 @@ jobs:
 {{- range $_, $cfScheduler := slice "diego" "eirini" }}
 
 - name: deploy-{{ $cfScheduler }}
-  max_in_flight: 1 # Use concourse queue which doesn't timeout
+  #max_in_flight: 1 # Use concourse queue which doesn't timeout
```

Comment:
Unused code should be removed, or have a comment on why it's left commented out.

Reply:
thanks for reminding, I forgot to update the comment :sweat_smile:  Done in https://github.com/cloudfoundry-incubator/kubecf/pull/598/commits/ce37a304fa3738f8062bb3b3393d0f7c2bfee77d , I would keep it around if we want to re-enable it in the future and put a cap on the parallel jobs so everyone is aware, but no strong preference, I can also drop it

Reply:
Thanks!